pkg/provider/kubernetes/crd/traefikio/v1alpha1/ingressroutetcp.go

```diff
@@ -28,9 +28,11 @@ type RouteTCP struct {
 	Match string `json:"match"`
 	// Priority defines the router's priority.
 	// More info: https://doc.traefik.io/traefik/v3.2/routing/routers/#priority_1
+	// +kubebuilder:validation:Minimum=0
```

Comment:
```suggestion
	// +kubebuilder:validation:Maximum=9223372036854774807
```

There is no enforcement on the minimum value and negative value are allowed, but the max is `const maxUserPriority = math.MaxInt - 1000`.